changelog-tool/main.pony

```diff
@@ -171,6 +180,24 @@ actor Main
       err("unable to perform release preparation")
     end
 
+  fun cmd_add(
+    filepath: FilePath,
+    filename: String,
+    section: String,
+    entry: String,
+    edit: Bool)
+  =>
+    try
+      edit_or_print(
+        filepath,
+        edit,
+	Changelog(parse(filepath, filename)?)?
```

Comment:
I'd suggest assigning this `Changelog(...) .> (...) .string()` expression to a variable before this function call. Otherwise it should be indented further.

Reply:
Again, my vim configs were messed up. Just formatted like the command above 👍